### DISCUSSION TITLE
LInk argo rollouts to ArgoCD URL

### DISCUSSION THREAD

**Discussion by OteemoSanjay** [argoproj/argo-rollouts#4017] [General]
hi

We are currently using ArgoCD for GitOps and it is deployed into k8.
Lets say our URL is :  https://deployment.dev.company.com

This works fine for DEV and we have multiple repos syncing to different NS (namespaces) in K8.

We`ve installed Argo Rollout and with the dashboard we can have port forwardning to the 3100 port.
But how do we link argo rollout to the URL after our argoCD URL.

So sticking with the example.   Rollouts would be available as:  https://deployments.dev.company.com/rollouts

I can expose the SVC and create a VS, but I want it linked to the ArgoCD URL

thanks in advance

rgds sanj

**Comment by kostis-codefresh** [comment]
I don't see how this question is specific to Argo Rollouts.

It is standard Kubernetes process. You need to check the ingress support of your ingress provider or your DNS settings from your DNS provider.

Basically expose Argo Rollouts like any other Kubernetes application.
